source/fundamentals/aggregation.txt

```diff
@@ -179,6 +179,59 @@ The above aggregation should produce the following results:
    {"_id": 4, "count": 2}
    {"_id": 5, "count": 1}
 
+Explain Aggregation Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To explain the execution plan of the operation, append the :java-docs:`explain()
+<apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`
+method containing a :java-core-api:`verbosity level <com/mongodb/ExplainVerbosity>` to the ``aggregates()`` method.
```

Comment:
```suggestion
method of the ``FindIterable`` class containing a :java-core-api:`verbosity level <com/mongodb/ExplainVerbosity>` to the ``aggregates()`` method.
```

Issue (possibly suggestion): Could the class the method belongs to be stated? I don't remember if we agreed on this as a team, but it feels jarring to have a method stated without the class it belongs to.